drive/snippets/drive_v3/file_snippets/move_file_to_folder.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START drive_move_file_to_folder]
+
+/**
+ * Move a file to specified folder
+ * @param{string} realFileId Id of the file to move
+ * @param{string} realFolderId Id of the folder to move
+ * @return{obj} file status
+ * */
+async function moveFileToFolder(realFileId, realFolderId) {
```

Comment:
Same comments as export_pdf -- align the auth comments & remove the fileId/folderId reassignments